Updates to choosing servos page (#395)
gamemanual0/gm0: source/docs/power-and-electronics/servo-guide/choosing-servo.rst

The angle over which a non-continuous :term:`servo <Servo>` can rotate while retaining position feedback is called the range. When choosing a servo, it is important to make sure you have enough range for the movement you need.

By default, the FTC Control Hub and FTC Expansion Hub output 600-2400 microsecond signals. However, this range can be expanded to 500-2500 microseconds. When choosing a servo, it is important to make sure that its range will be usable for your application inside of 500-2500 microseconds.

.. note::

   The default 600-2400 range of the FTC Expansion Hub and FTC Control Hub can make it appear that popular servos like the goBILDA Dual Mode servos and REV Smart Robot Servo have less range then advertised. You can use the following code to expand the range to 500-2500 microseconds.

   .. code-block:: java

      ServoImplEx servo = hardwareMap.get(ServoImplEx.class, "myservo");
      ...
      servo.setPwmRange(new PwmRange(500, 2500));

Cost
----

:term:`Servos <Servo>` range from cheap $7 :term:`servos <Servo>` for light applications, all the way up to some Hitec or Savox :term:`servos <Servo>` for close to $200.

> By far the best bang for your buck :term:`servos <Servo>` out there are going to be **goBILDA dual mode** and **REV SRS** servos. In addition, the **Andymark High Torque/Speed** servo has on paper specifications making it the best bang for your buck servo, but at the time of writing has not been released and tested.

Reviewer: ```suggestion
By far the best bang for your buck :term:`servos <Servo>` out there are going to be **goBILDA dual mode** and **REV SRS** servos. In addition, the **Andymark High Torque/Speed** servos on paper are the best bang for your buck servo, but at the time of writing have not been released and tested.
```

The last big player in the :term:`servo <Servo>` market in FTC is Savox, which produces great mid-high range :term:`servos <Servo>` (think $60-$100+). They are made with titanium :term:`gears <Gear>` (close to bulletproof) and are **fast**. Savox :term:`servos <Servo>` are mostly brushless and coreless, so they do tend to scream a little under load, but they’re definitely worth it if your budget allows for it.

Recommended Servos
------------------

> GM0 no longer recommends low priced servos. Due to their low strength, they end up requiring multiple replacements over time, negating any cost benefits.

Reviewer: ```suggestion
We no longer recommend low priced servos. Due to their low strength, they end up requiring multiple replacements over time, negating any cost benefits.
```
